olive/snpe/data_loader.py

```diff
@@ -163,15 +164,15 @@ def load_data(self) -> Tuple[str, str, np.ndarray]:
 class SNPERandomDataLoader(SNPEDataLoader):
     def __init__(
         self,
-        io_config: dict,
+        io_config: IOConfig,
```

Comment:
>IOConfig [](http://example.com/codeflow?start=19&length=8)

better to keep original dict to keep the logic simpler instead of accept IOConfig and do conversion using dict()